### PR TITLE
Feat/vergoeding form

### PR DESCRIPTION
**Pull request by GoGoGadgetMusic**

### CODE REVIEW

File: src/components/vergoeding.svelte.html [hunk 1]
```diff
@@ -1,47 +1,121 @@
...
-        <label>
-          {{ question.label }}
-          <input type="radio" value="y" name="{{ question.key }}" bind:group="question.checked" on:change="checkValidity()"> ja
-          <input type="radio" value="n" name="{{ question.key }}" bind:group="question.checked" on:change="checkValidity()"> nee
-        </label>
-      </li>
-      {{ /each }}
-    {{ /if }}
-    </ol>
+    <h2 class="toggle">{{ vergoeding.title }}</h2>
```
Comment (celinekurpershoek) [inline src/components/vergoeding.svelte.html]: The title is now defined in the step

File: src/components/ww.svelte.html [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
-    <li>
-      <label>
-        {{ question.label }}
+      {{ #each ww.questions as question }}
+      <li>
+          {{ #if question.key === "einddatum" }}
+          <label for="einddatum">{{ question.label }}</label>
+          <input id="einddatum" type="date" name="{{ question.key }}">
+          {{ else }}
+          <fieldset>
```
Comment (celinekurpershoek) [inline src/components/ww.svelte.html]: 🙀 fieldsets === no flexbox. But ok for now :)

File: src/components/vergoeding.svelte.html [hunk 3]
```diff
@@ -1,47 +1,121 @@
...
+                </div>
+
+                <div class="form-field">
+                    <select id="reason" on:change="selectReason(event.target.value)">
+                        <option disabled selected>Reden van ontslag</option>
+                        {{ #each vergoeding.reasons as reason, index }}
+                            <option value="{{ index }}">{{ reason.label }}</option>
+                        {{ /each }}
+                    </select>
+                    <label for="reason" class="a11y-sr-only">Reden van ontslag</label>
```
Comment (celinekurpershoek) [inline src/components/vergoeding.svelte.html]: `Reden van ontslag` Shouldn't this label come from dato?

File: src/components/vergoeding.svelte.html [hunk 4]
```diff
@@ -1,47 +1,121 @@
...
+                    <input id="fee"
+                           type="number" 
+                           placeholder="{{ questions.fee.label }}"
+                           disabled="{{ !showFeeInput }}">
+                    <label for="fee" class="a11y-sr-only" >{{ questions.fee.label }}</label>
+                </div>
+
+                <div class="form-field">
+                    <select id="reason" on:change="selectReason(event.target.value)">
+                        <option disabled selected>Reden van ontslag</option>
```
Comment (celinekurpershoek) [inline src/components/vergoeding.svelte.html]: `Reden van ontslag` shouldn't this come from dato?

File: src/components/ww.svelte.html (hunk 2 above)
Comment (GoGoGadgetMusic) [reply src/components/ww.svelte.html]: a11y 🤘

**Pull request merged**